src/main/java/net/sf/jabref/logic/importer/fetcher/MedlineFetcher.java

```diff
@@ -0,0 +1,211 @@
+
+    @Override
+    public Parser getParser() {
+        return new MedlineImporter();
+    }
+
+    @Override
+    public void doPostCleanup(BibEntry entry) {
+        entry.clearField("journal-abbreviation");
+        entry.clearField("status");
+        entry.clearField("copyright");
+    }
+
+    @Override
+    public List<BibEntry> performSearch(String query) throws FetcherException {
+        List<BibEntry> entryList = new LinkedList<>();
+
+        if (query.isEmpty()) {
+            return Collections.emptyList();
+        } else {
+            String searchTerm = replaceCommaWithAND(query);
+
+            //searching for pubmed ids matching the query
+            List<String> idList = getPubMedIdsFromQuery(searchTerm);
+
+            if (idList.isEmpty()) {
+                LOGGER.info(Localization.lang("No results found."));
+                return Collections.emptyList();
+            }
+            if (numberOfResultsFound > 50) {
+                LOGGER.info(numberOfResultsFound + " results found. Only 50 relevant results will be fetched by default.");
+            }
+
+            //pass the list of ids to fetchMedline to download them. like a id fetcher for mutliple ids
+            entryList = fetchMedline(idList);
+
+            return entryList;
+        }
+    }
+
+    private URL createSearchUrl(String term) throws URISyntaxException, MalformedURLException {
+        term = replaceCommaWithAND(term);
+        URIBuilder uriBuilder = new URIBuilder("http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi");
+        uriBuilder.addParameter("db", "pubmed");
+        uriBuilder.addParameter("sort", "relevance");
+        uriBuilder.addParameter("retmax", String.valueOf(NUMBER_TO_FETCH));
+        uriBuilder.addParameter("term", term);
+        return uriBuilder.build().toURL();
+    }
+
+    /**
+     * Fetch and parse an medline item from eutils.ncbi.nlm.nih.gov.
+     * The E-utilities generate a huge XML file containing all entries for the ids
+     *
+     * @param ids A list of IDs to search for.
+     * @return Will return an empty list on error.
+     */
+    private List<BibEntry> fetchMedline(List<String> ids) throws FetcherException {
+        try {
+            //Separate the IDs with a comma to search multiple entries
+            URL fetchURL = getURLForID(String.join(",", ids));
+            URLConnection data = fetchURL.openConnection();
+            ParserResult result = new MedlineImporter().importDatabase(
+                    new BufferedReader(new InputStreamReader(data.getInputStream(), StandardCharsets.UTF_8)));
+            if (result.hasWarnings()) {
+                LOGGER.warn(result.getErrorMessage());
+            }
+            List<BibEntry> resultList = result.getDatabase().getEntries();
+            resultList.forEach(this::doPostCleanup);
+            return resultList;
+        } catch (URISyntaxException | IOException e) {
+            throw new FetcherException(e.getLocalizedMessage(), e);
```

Comment:
The messages of (most) API exceptions are not localized. Thus, calling `e.getLocalizedMessage()` will always return English messages. Provide an own localized message, please.